### ISSUE TITLE
Python UUID returned as Python String

### ISSUE THREAD

**Issue by polsm91** [bsc-dd/hecuba#114]
Hfetch accepts UUIDs as bytes and uuids of the module UUID. However, when they are accessed the UUIDs are returned as Strings. 
The UUIDs should be returned as objects of the module UUID.

**Comment by polsm91** [comment]
The UnitParser builds UUIDs as String, we should return them as ByteArray or UUIDs:
```C++
    PyObject *uuid_module =  PyImport_ImportModule("UUID");
   // Call uuid_module . UUID()
```

See    https://docs.python.org/3/c-api/object.html

**Comment by polsm91** [comment]
Closed by 78f9a69c39091c54cb4548ee1b5492fcdd5a0302
